### PR TITLE
certificates ui

### PR DESCRIPTION
**Pull request by psantos1113**
create the UI to import certificates into Alert.

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -297,36 +297,6 @@ createDataBackUp(){
     fi
 }
 
-checkVolumeDirectories() {
```
Comment (jamesrichard91) [inline docker-entrypoint.sh]: Was this not needed anymore?

File: src/main/js/store/actions/certificates.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    CERTIFICATES_FETCHED_ALL,
+    CERTIFICATES_FETCHING_ALL,
+    CERTIFICATES_SAVE_ERROR,
+    CERTIFICATES_SAVED,
+    CERTIFICATES_SAVING
+} from "store/actions/types";
+
+import * as RequestUtilities from 'util/RequestUtilities'
+import { verifyLoginByStatus } from "./session";
+import { fetchUsers } from "./users";
```
Comment (jamesrichard91) [inline src/main/js/store/actions/certificates.js]: Can you make these absolute paths?

File: src/main/js/dynamic/loaded/certificates/CertificatesPage.js [hunk 3]
```diff
@@ -0,0 +1,197 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from "prop-types";
+import { clearCertificateFieldErrors, deleteCertificate, fetchCertificates, saveCertificate } from "store/actions/certificates";
+import ConfigurationLabel from "component/common/ConfigurationLabel";
+import TableDisplay from "field/TableDisplay";
+import TextInput from "field/input/TextInput";
+import TextArea from "field/input/TextArea";
+import ReadOnlyField from "field/ReadOnlyField";
```
Comment (bamandel) [inline src/main/js/dynamic/loaded/certificates/CertificatesPage.js]: These should be single quotes

File: docker-entrypoint.sh (hunk 1 above)
Comment (psantos1113) [reply docker-entrypoint.sh]: no with the database container defining the volume PGDATA and docker-compose isn't going to be supported we don't have to verify that we can write to the volume. The encryption data is stored as secrets.  The accumulator has the last runtime stored in the database.  So we should be good here to remove this.

File: src/main/js/store/actions/certificates.js (hunk 2 above)
Comment (psantos1113) [reply src/main/js/store/actions/certificates.js]: will do

**Pull request merged**